config.json

```diff
@@ -1,848 +1,863 @@
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 3,
+        "topics": [
+          "arrays",
+          "math"
+        ]
+      },
+      {
+        "slug": "scrabble-score",
+        "name": "Scrabble Score",
+        "uuid": "54a0753e-c161-4dfd-a7b0-02dc7a923d82",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 3,
+        "topics": [
+          "maps",
+          "strings"
+        ]
+      },
+      {
+        "slug": "all-your-base",
+        "name": "All Your Base",
+        "uuid": "5a839eb7-cc4b-48eb-bd04-57ba6e77eae8",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 4,
+        "topics": [
+          "algorithms",
+          "integers",
+          "loops",
+          "math"
+        ]
+      },
+      {
+        "slug": "robot-simulator",
+        "name": "Robot Simulator",
+        "uuid": "ed2e467d-621d-42de-afdd-f9b49efbc263",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 4,
+        "topics": [
+          "classes",
+          "conditionals",
+          "loops",
+          "pairs",
+          "strings",
+          "variables"
+        ]
+      },
+      {
+          "slug": "bank-account",
+          "name": "Bank Account",
+          "uuid": "56cbac7e-8bd8-4cd5-bfc5-0bc512e95de0",
+          "practices": [],
+          "prerequisites": [],
+          "difficulty": 8,
+          "topics": [
+              "basics",
+              "classes",
+              "conditionals",
+              "numbers",
+              "threads"
+          ]
+      },
+      {
+        "slug": "darts",
+        "name": "Darts",
+        "uuid": "e8ad3859-9999-4a36-900d-d82fa6e74b08",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 2,
+        "topics": [
+            "basics",
+            "bools",
+            "conditionals",
+            "numbers"
+        ]
+      },
+      {
+        "slug": "largest-series-product",
+        "name": "Largest Series Product",
+        "uuid": "bf25f471-20d9-4f41-abf7-5d0dc5ad2d87",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 4,
+        "topics": [
+            "bools",
+            "conditionals",
+            "lists",
+            "loops",
+            "numbers",
+            "strings"
+        ]
+      },
+      {
+        "slug": "isbn-verifier",
+        "name": "Isbn Verifier",
+        "uuid": "5fcc6fd8-1dc7-4493-970a-731c7602b229",
+        "practices": [],
+        "prerequisites":[],
+        "difficulty": 1,
+        "topics": [
+          "conditionals",
+          "loops",
+          "strings"
+        ]
+      },
+      {
+        "slug": "simple-linked-list",
+        "name": "Simple linked list",
+        "uuid": "ffccabe9-9779-4914-9cff-c6f5696c8afe",
+        "practices": ["pointers"],
+        "prerequisites":[],
+        "difficulty": 4,
+        "status": "wip",

```

Comment:
Should we publish with `wip` or `beta`?

Reply:
Students won't see the exercise with `wip`.

Reply:
That is how I understood that. Should we publish as beta, so more people can see it? Or should we keep it to the (cpp?) maintainers first? that is what wip would do, right?

Reply:
I think it is fine to have this as `beta`